HISTORY.md

```diff
@@ -1,3 +1,16 @@
+0.16.0 / 2021-04-02
+--------------------
+- restore #404, add get_correction_from_cmt to corrections_services.py (#409)
```

Comment:
I think you can better use the title of #404 and use both #404 and #409 as reference

Reply:
Okay, I change it